zoninator.php

```diff
@@ -1128,11 +1128,11 @@ function get_zones_for_post( $post_id ) {
 
 	function get_zones( $args = array() ) {
 
-		$args = wp_parse_args( $args, array(
+		$args = apply_filters( 'zoninator_terms_query_args', wp_parse_args( $args, array(
```

Comment:
@bcampeau Makes sense.

@damonsharp How do you plan on using the proposed filter? Can the core filters in `get_terms` do what you need to?